backend/ctgov/api/urls.py

```diff
@@ -24,4 +24,5 @@
     path("trials_dashboard", TrialsDashboardApiView.as_view()),
     path("search_results_export", SearchResultsExportApiView.as_view()),
     path("trial_timelines", TrialTimelinesApiView.as_view()),
+    path("healthz", include("health_check.urls")),
```

Comment:
This health check endpoint is required for running django behind the AWS load balancer.